src/overlays/actors/ovl_En_Talk/z_en_talk.h

```diff
@@ -14,4 +14,6 @@ typedef struct EnTalk {
 
 extern const ActorInit En_Talk_InitVars;
 
+#define GET_TEXT_ID(this) ((this->actor.params & 0x3F) + 0x1C00)
```

Comment:
Since this macro is specific to this actor, it should be prefixed with the actor's name to avoid confusion in other places
```suggestion
#define EN_TALK_GET_TEXT_ID(this) ((this->actor.params & 0x3F) + 0x1C00)
```